openshift_tools/ircbot/trello/trello.py

```diff
@@ -128,6 +150,97 @@ def get(self):
                                                  members)
         return results
 
+    def report(self, email=None, move=False):
+        """Generate reports"""
+        payload = self.report_payload()
+        print(payload)
+        _env_email = os.environ.get("trello_report_email_addresses", None)
```

Comment:
Why duplicate this when it should already be available in `self. email_addresses`?

Reply:
self.email_address is passed in as CLI, which overrides environment variable. CLI args aren't available in sopel use case so we fallback to env var.